cmake/SSGCommon.cmake

```diff
@@ -295,7 +295,7 @@ macro(ssg_build_ansible_playbooks PRODUCT)
 endmacro()
 
 macro(ssg_build_remediations PRODUCT)
-    message(STATUS "Scanning for dependencies of ${PRODUCT} fixes (bash, ansible, puppet, anaconda, ignition and kubernetes)...")
+    message(STATUS "Scanning for dependencies of ${PRODUCT} fixes (bash, ansible, puppet, anaconda, ignition, kubernetes and osbuild_blueprint)...")
```

Comment:
Had just a quick look, but isn't `osbuild_blueprint` a bit too long?
I see that in https://github.com/OpenSCAP/openscap/pull/1749 the fix type is just called blueprint.